beacon_node/lighthouse_network/src/types/globals.rs

```diff
@@ -255,7 +268,15 @@ impl<E: EthSpec> NetworkGlobals<E> {
         let keypair = libp2p::identity::secp256k1::Keypair::generate();
         let enr_key: discv5::enr::CombinedKey = discv5::enr::CombinedKey::from_secp256k1(&keypair);
         let enr = discv5::enr::Enr::builder().build(&enr_key).unwrap();
-        NetworkGlobals::new(enr, metadata, trusted_peers, false, config, spec)
+        NetworkGlobals::new(
+            enr,
+            metadata,
+            trusted_peers,
+            false,
+            config,
+            spec.number_of_columns,
```

Comment:
should this be custody groups?
```suggestion
            spec.number_of_custody_groups,
```